spec/rspec/matchers/built_in/has_spec.rb

```diff
@@ -65,15 +65,15 @@ def obj_with_block_method.has_some_stuff?; yield; end
     def o.has_some_stuff?; false; end
     expect {
       expect(o).to have_some_stuff
-    }.to fail_with("expected #has_some_stuff? to return true, got false")
+    }.to fail_with("expected `#{o.inspect}.has_some_stuff?` to return true, got false")
```

Comment:
I'm a bit out of context. Where does the `inspect` part come from? I would expect the message to be "expected `o.has_some_stuff?` to return true, got false". Is that something you were referring to in "produce a better output"?

Reply:
Its because `actual` e.g `o` is being printed by the matcher

Reply:
Got it. I didn't parse it at first.